packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/template/ArrayFieldDescriptionTemplate.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {
+  getTemplate,
+  getUiOptions,
+  ArrayFieldDescriptionProps,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+} from '@rjsf/utils'
+
+/** The `ArrayFieldDescriptionTemplate` component renders a `DescriptionFieldTemplate` with an `id` derived from
+ * the `idSchema`.
+ *
+ * Identical to @rjsf/core ArrayFieldDescriptionTemplate except this doesn't render nothing if there is no description,
```

Comment:
"except this doesn't render nothing if there is no description" -> "except this renders something if there is no description"?